examples/pytorch/rgcn-hetero/entity_classify_mb.py

```diff
@@ -86,6 +87,12 @@ def main(args):
         labels = labels.to(device)
         embed_layer = embed_layer.to(device)
 
+    if args.num_workers <= 0:
```

Comment:
why not support args.num_workers = 0 which is existing behavior?

Reply:
In fact, when the `dataloader` is created with `args.num_workers=0`:
```
    dataloader = dgl.dataloading.DataLoader(
       . . .
        num_workers=args.num_workers,
    )
```
and 
```
with dataloader.enable_cpu_affinity():
```
is used, we get the following exception:
```
  File "/usr/local/lib/python3.10/dist-packages/dgl/dataloading/dataloader.py", line 1118, in enable_cpu_affinity
    raise Exception(
Exception: ERROR: affinity should be used with at least one DL worker
```